tensorflow_addons/image/README.md

```diff
@@ -8,6 +8,8 @@
 | distort_image_ops | @WindQAQ | windqaq@gmail.com |
 | filters | @Mainak431 | mainakdutta76@gmail.com |
 | transform_ops | @mels630 | mels630@gmail.com | 
+| translate_ops | @sayoojbk	| sayoojbk@gmail.com |
+| translations_to_projective_transforms | @sayoojbk	| sayoojbk@gmail.com |
 
 ## Components 
 | Submodule  | Image Processing Function |  Reference  |
```

Comment:
Have to add some rows to this table:
```
| translate_ops | translate | |
| translate_ops | translations_to_projective_transforms | |
```
Please put them alphabetically.

Reply:
I have added them alphabetically and removed the line containing the name I added.

Reply:
Oops, I mean we only have to remove this line
```
| translations_to_projective_transforms | @sayoojbk	| sayoojbk@gmail.com |
```